himanshu/postgresAssignment/src/controllers/resultContoller.ts

```diff
@@ -0,0 +1,157 @@
+const resultServices = require('../services/resultServices.ts');
+
+exports.addMarks = async (ctx :any) => {
+  const object = ctx.request.body;
+  if (object.studentId == null || object.subjectId == null || object.marks == null) {
```

Comment:
we can use joi or ajv for validation 
https://joi.dev/api/?v=17.4.0 

just try it out no need to make any changes